radix-engine-tests/tests/blueprints/non_fungible/src/lib.rs

```diff
@@ -382,17 +392,17 @@ mod non_fungible_test {
 
             // read singleton bucket
             let singleton = bucket.take(1);
-            let _: Sandwich = singleton.non_fungible().data();
+            let _: Sandwich = singleton.as_non_fungible_bucket().non_fungible().data();
 
             // read singleton vault
             let mut vault = Vault::with_bucket(singleton);
-            let _: Sandwich = vault.non_fungible().data();
+            let _: Sandwich = vault.as_non_fungible_vault().non_fungible().data();
 
             // read singleton proof
-            let proof = vault.create_proof();
-            let validated_proof = proof.validate_proof(vault.resource_address()).unwrap();
-            let _: Sandwich = validated_proof.non_fungible().data();
-            validated_proof.drop();
+            let proof = vault.create_proof().skip_checking();
+            assert_eq!(proof.resource_address(), vault.resource_address());
```

Comment:
Out of interest - why did we remove `validate_proof`? Or why did we change this line if we didn't remove it?

Reply:
Sorry, I don't really remember this specific line change. It's possibly caused by massive find & replace. :)